### PR TITLE
[OTel] Refactor cassandra instrumentation

### PR DESCRIPTION
**Pull request by CagriYonca**
Coverage: 96%

### CODE REVIEW

File: src/instana/instrumentation/cassandra_inst.py [hunk 1]
```diff
@@ -6,78 +6,103 @@
...
+            fn.add_callback(cb_request_finish, span, fn)
+            fn.add_errback(cb_request_error, span, fn)
+
+    @wrapt.patch_function_wrapper("cassandra.cluster", "Session.__init__")
+    def init_with_instana(
+        wrapped: Callable[..., object],
+        instance: Session,
+        args: Tuple[object, ...],
+        kwargs: Dict[str, Any],
+    ) -> Any:
```
Comment (pvital) [inline src/instana/instrumentation/cassandra_inst.py:101]:
```suggestion
    ) -> object:
```

**Review by pvital — CHANGES_REQUESTED** [review]
Very few requests :-D

File: src/instana/instrumentation/cassandra_inst.py (hunk 1 above)
Comment (CagriYonca) [reply src/instana/instrumentation/cassandra_inst.py]: Done

**Review by GSVarsha — APPROVED** [review]
LGTM!

**Pull request merged**